rules/import.js

```diff
@@ -5,5 +5,14 @@ module.exports = {
     'import/no-cycle': ['error', { maxDepth: Infinity }],
     'import/no-default-export': 'error',
     'import/no-self-import': 'error',
+    'import/no-extraneous-dependencies': [
+      'error',
+      {
+        devDependencies: false,
```

Comment:
> devDependencies: If set to false, then the rule will show an error when devDependencies are imported. 

Wouldn't this cause an error when we are using a devDependency from a test file unless we make an override in every repo's eslint config?

Reply:
Yeah, this might have some nicer defaults: https://github.com/ridedott/eslint-config/pull/1208/commits/4dd6e16d35678a419f3b0d10dd2a382448c7a7b6